crates/notary/server/src/domain.rs

```diff
@@ -16,4 +17,7 @@ pub struct InfoResponse {
     pub git_commit_hash: String,
     /// Current git commit timestamp of notary-server
     pub git_commit_timestamp: String,
+    /// Hardware attestation
```

Comment:
what happened to the changes in `openapi.yaml`? the response should still be updated